<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materialStorgeWidget</class>
 <widget class="QWidget" name="materialStorgeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>入库管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="pop_widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777214</width>
       <height>35</height>
      </size>
     </property>
     <widget class="QPushButton" name="pop_close_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>6</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pop_min_btn">
      <property name="geometry">
       <rect>
        <x>535</x>
        <y>6</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(0, 200, 255);</string>
      </property>
      <property name="text">
       <string>入库管理</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pop_cancle_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>540</y>
        <width>95</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pop_commit_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>540</y>
        <width>95</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>日期：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="use_Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>用途：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>物品名称：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>单位：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject_name_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="unit_Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="error_lab">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>381</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>有选项为空，请核实！</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject_Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>类别：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>运费：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="freight_Edit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>成本价格：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="pice_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Number_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="norms_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>物品规格：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
